tests/integration/new_relations/test_new_relations.py

```diff
@@ -220,7 +220,9 @@ async def test_filter_out_degraded_replicas(ops_test: OpsTest):
             assert data is None
 
     await start_machine(ops_test, machine)
-    await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status="active", timeout=200)
+    await ops_test.model.wait_for_idle(
+        apps=[DATABASE_APP_NAME], status="active", timeout=200, raise_on_error=False
+    )
```

Comment:
Unrelated, but the charm seems to frequently error out when the container is started: https://github.com/canonical/postgresql-operator/actions/runs/12847342758/job/35866184943